pkg/intermediate/types.go

```diff
@@ -45,6 +45,12 @@ type AggregationFlowRecord struct {
 	// false when creating new AggregationFlowRecord. Setting and utilizing this
 	// field is upto the user and not used in go-ipfix library code.
 	areExternalFieldsFilled bool
+	// isIPv4 indicates whether the source and destination addresses are IPv4 or
+	// IPv6 in the aggregated flow record.
+	isIPv4 bool
+	// isExporterIPv4 indicates whether the exporter address of the received flow
+	// aggregator is IPv4 or IPv6.
+	isExporterIPv4 bool
```

Comment:
We should either make these two fields public or have methods to access them if we want to use at Antrea side.

Reply:
Agreed. Missed it. Will add Get interface for these metadata fields and they are needed.